routes/router.js

```diff
@@ -50,6 +51,9 @@ module.exports = (routesJson, config) => {
             , routeInfo
             , res = resIn;
 
+        // add .setStatus to response
+        res.setStatus = setStatus();
```

Comment:
This would change per the comment below to just `res.status = setStatus` since setStatus will no longer need to be executed to fill its function.